tests/reg_tests/test_separation.py

```diff
@@ -0,0 +1,342 @@
+# built-ins
+import unittest
+import numpy as np
+import os
+import copy
+
+# MACH classes
+from adflow import ADFLOW
+from adflow import ADFLOW_C
+
+# import the testing utilities
+from parameterized import parameterized_class
+from reg_default_options import adflowDefOpts
+from reg_aeroproblems import ap_naca0012_separation
+import reg_test_classes
+
+baseDir = os.path.dirname(os.path.abspath(__file__))
+gridFile = os.path.join(baseDir, "../../input_files/naca0012_L3_SEP.cgns")
+test_params = [
+    {
+        "name": "naca0012_rans_2D_surfvec_ks",
+        "ref_file": "separation_tests_surfvec_ks.json",
+        "aero_prob": ap_naca0012_separation,
+        "N_PROCS": 2,
+        "options": {
```

Comment:
Most of the options are common across the different tests, so I would recommend adding a common options dictionary in `setUp` and defining the options that are different in `test_params`